src/components/about/Header.jsx

```diff
@@ -1,7 +1,14 @@
 import React from "react";
 
 const Header = () => {
-  return <div>Header</div>;
+  return (
+    <div className="mt-8 ml-8">
+      <h1 className="text-4x1 font-bold tracking-widest text-blue-700">
```

Comment:
The Header currently does look like the Figma, make sure to use the current font, color, and font-size, etc. 
Also go back and look at the issue, because you didn't add props